client/src/components/CommentBody/CommentBody.css

```diff
@@ -1,19 +1,66 @@
+body {
+  text-align: center;
+}
+
 .CommentBody {
```

Comment:
I think this class name should be `Comment__body`, per the naming convention in the readme